js/signup.js

```diff
@@ -0,0 +1,49 @@
+$(document).ready(function() {
+    $('#sign-up-form').bootstrapValidator({
+        // To use feedback icons, ensure that you use Bootstrap v3.1.0 or later
+        feedbackIcons: {
+            valid: 'glyphicon glyphicon-ok',
+            invalid: 'glyphicon glyphicon-remove',
+            validating: 'glyphicon glyphicon-refresh'
+        },
+        fields: {
+            username: {
+                validators: {
+                        stringLength: {
+                        min: 2,
+                    },
+                        notEmpty: {
+                        message: 'Please enter a valid username'
+                    }
+                }
+            },
+        	 email: {
+                validators: {
+                    notEmpty: {
+                        message: 'Please supply your email address'
+                    },
+                    emailAddress: {
+                        message: 'Please supply a valid email address'
+                    }
+                }
+            },
+
+        	password: {
+                    validators: {
+                        identical: {
+                            field: 'confirmPassword',
+                            message: 'Confirm your password below - type same password please'
```

Comment:
I am not sure if this is the right field and message. Did you mix it with the one below?